src/main/java/com/faforever/client/mod/ModService.java

```diff
@@ -337,6 +337,10 @@ public CompletableTask<Void> uploadMod(Path modPath) {
     ModUploadTask modUploadTask = modUploadTaskFactory.getObject();
     modUploadTask.setModPath(modPath);
 
+    // retrieve information from the mod_info.lua to send to the API
+    ModVersion modVersion = extractModInfo(modPath);
+    modUploadTask.setRepositoryURL(modVersion.mod().repositoryURL());
```

Comment:
This can be moved into the ModUploadTask itself

Reply:
Processed in https://github.com/FAForever/downlords-faf-client/pull/3192/commits/26166cbea6053b7adadf9a7e0d04725516fd7942